workspace/notebook/py_sb_horizon_harmonised_nsip_project.json

```diff
@@ -431,30 +600,51 @@
 					"collapsed": false
 				},
 				"source": [
-					"#Establish aggregated view of regions\n",
-					"horizon_region = horizon_data.groupBy(\"caseid\",\"IngestionDate\").agg(collect_list(\"region\").alias(\"regions\"))\n",
-					"horizon_data = horizon_data.join(horizon_region, on=[\"caseid\",\"IngestionDate\"], how=\"inner\").drop(\"region\")\n",
+					"try:\n",
+					"    #Establish aggregated view of regions\n",
+					"    horizon_region = horizon_data.groupBy(\"caseid\",\"IngestionDate\").agg(collect_list(\"region\").alias(\"regions\"))\n",
+					"    horizon_data = horizon_data.join(horizon_region, on=[\"caseid\",\"IngestionDate\"], how=\"inner\").drop(\"region\")\n",
+					"\n",
 					"\n",
+					"    #Build nsipOfficerIds \n",
+					"    horizon_officer = horizon_data.groupBy(\"caseid\",\"IngestionDate\").agg(collect_list(\"nsipOfficerIds\").alias(\"nsipOfficerIds\"))\n",
 					"\n",
-					"#Build nsipOfficerIds \n",
-					"horizon_officer = horizon_data.groupBy(\"caseid\",\"IngestionDate\").agg(collect_list(\"nsipOfficerIds\").alias(\"nsipOfficerIds\"))\n",
+					"    horizon_data = horizon_data.drop(\"nsipOfficerIds\")\n",
+					"    horizon_data = horizon_data.join(horizon_officer, on=[\"caseid\",\"IngestionDate\"], how=\"inner\")\n",
```

Comment:
What are these changes for? I thought this PR is just for the logging?

Reply:
Hmm the changes are still there after the rebase. Sorry to be petty but i think we need to double check these.